// react-payment-logos/dist/logo/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Visa = exports.Unionpay = exports.Paypal = exports.Mir = exports.Mastercard = exports.Maestro = exports.Jcb = exports.Hipercard = exports.Hiper = exports.Elo = exports.Discover = exports.Diners = exports.Amex = exports.Alipay = void 0;
var Alipay_1 = require("./Alipay");
Object.defineProperty(exports, "Alipay", { enumerable: true, get: function () { return __importDefault(Alipay_1).default; } });
var Amex_1 = require("./Amex");
Object.defineProperty(exports, "Amex", { enumerable: true, get: function () { return __importDefault(Amex_1).default; } });
var Diners_1 = require("./Diners");
Object.defineProperty(exports, "Diners", { enumerable: true, get: function () { return __importDefault(Diners_1).default; } });
var Discover_1 = require("./Discover");
Object.defineProperty(exports, "Discover", { enumerable: true, get: function () { return __importDefault(Discover_1).default; } });
var Elo_1 = require("./Elo");
Object.defineProperty(exports, "Elo", { enumerable: true, get: function () { return __importDefault(Elo_1).default; } });
var Hiper_1 = require("./Hiper");
Object.defineProperty(exports, "Hiper", { enumerable: true, get: function () { return __importDefault(Hiper_1).default; } });
var Hipercard_1 = require("./Hipercard");
Object.defineProperty(exports, "Hipercard", { enumerable: true, get: function () { return __importDefault(Hipercard_1).default; } });
var Jcb_1 = require("./Jcb");
Object.defineProperty(exports, "Jcb", { enumerable: true, get: function () { return __importDefault(Jcb_1).default; } });
var Maestro_1 = require("./Maestro");
Object.defineProperty(exports, "Maestro", { enumerable: true, get: function () { return __importDefault(Maestro_1).default; } });
var Mastercard_1 = require("./Mastercard");
Object.defineProperty(exports, "Mastercard", { enumerable: true, get: function () { return __importDefault(Mastercard_1).default; } });
var Mir_1 = require("./Mir");
Object.defineProperty(exports, "Mir", { enumerable: true, get: function () { return __importDefault(Mir_1).default; } });
var Paypal_1 = require("./Paypal");
Object.defineProperty(exports, "Paypal", { enumerable: true, get: function () { return __importDefault(Paypal_1).default; } });
var Unionpay_1 = require("./Unionpay");
Object.defineProperty(exports, "Unionpay", { enumerable: true, get: function () { return __importDefault(Unionpay_1).default; } });
var Visa_1 = require("./Visa");
Object.defineProperty(exports, "Visa", { enumerable: true, get: function () { return __importDefault(Visa_1).default; } });
